webroot/src/components/Licensee/LicenseeList/LicenseeList.ts

```diff
@@ -236,7 +267,7 @@ class LicenseeList extends Vue {
             this.nextKey = '';
         }
 
-        if (this.isInitialFetchCompleted) {
+        if (!isInitialInProgress && this.isInitialFetchCompleted) {
```

Comment:
Is this new condition something we can test?

Reply:
Good idea. Done.